Book_my_show_schema.txt

```diff
@@ -0,0 +1,86 @@
+create table Users(
+userid int primary key identity(1,1),
+Name varchar(50) not null,
+phone_number nvarchar(15),
+email varchar(254) unique,
+password varchar(50) not null);
+
+
+create table Movies(
+Movieid int primary key identity(1,1),
+Genre varchar(20) not null,
+Duration Time not null,
+language varchar(50),
+Realesedate Date not null);
+
+
+create table addresses(
+addressid int primary key identity(1,1),
+StreetAddress VARCHAR(100),
+City VARCHAR(50),
+State VARCHAR(50),
+PostalCode VARCHAR(10),
+Country VARCHAR(50));
+
+
+create table Theatres(Theatreid int primary key identity(1,1),
+addressid int not null,
+totalscreens int not null,
+foreign key(addressid) references addresses(addressid));
+
+
+CREATE TABLE Screens (
+ScreenID INT PRIMARY KEY IDENTITY(1,1),               
+ScreenNumber NVARCHAR(10) NOT NULL, 
+Capacity INT NOT NULL,        
+ScreenType NVARCHAR(50), 
+Theatreid int not null,
+totalshows int not null,
+foreign key(Theatreid) references theatres(theatreid));
+
+
+create table Shows(ShowID INT PRIMARY KEY IDENTITY(1,1),
+ScreenID INT NOT NULL,                           
+MovieID INT NOT NULL,                            
+ShowTime DATETIME NOT NULL,                                        
+SeatsAvailable INT NOT NULL,  
```

Comment:
relevance and logic for update for this field

Reply:
to track the number of seats left for a specific show.It ensures users can book only if there are enough seats available.
 It should be updated by subtracting the number of seats booked for that specific show from the current available seats